<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,0,1,5">
    <item>
     <widget class="QFrame" name="multimedialPanel">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="WaveformView" name="waveform" native="true"/>
       </item>
       <item>
        <widget class="VideoPlayer" name="videoPlayer"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="playbackControlsPanel">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="play">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timer">
         <property name="text">
          <string>0:00:00,000</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Selected subtitle:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SubtitleDialogEditor" name="subtitleDialogEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SubtitleListView" name="subtitleList"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_subtitles"/>
    <addaction name="actionOpen_subtitles"/>
    <addaction name="actionRemove_selected_subtitle"/>
    <addaction name="actionDetach_video"/>
    <addaction name="actionOpen_video"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_subtitles">
   <property name="text">
    <string>New subtitles</string>
   </property>
  </action>
  <action name="actionOpen_subtitles">
   <property name="text">
    <string>Open subtitles</string>
   </property>
  </action>
  <action name="actionRemove_selected_subtitle">
   <property name="text">
    <string>Remove selected subtitle</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDetach_video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detach video</string>
   </property>
  </action>
  <action name="actionOpen_video">
   <property name="text">
    <string>Open video</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SubtitleDialogEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>ui_controllers/subtitle_dialog_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WaveformView</class>
   <extends>QWidget</extends>
   <header>ui_controllers/waveform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SubtitleListView</class>
   <extends>QTableView</extends>
   <header>ui_controllers/subtitle_list_view.h</header>
  </customwidget>
  <customwidget>
   <class>VideoPlayer</class>
   <extends>QOpenGLWidget</extends>
   <header>ui_controllers/video_player.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
